packages/@aws-cdk/aws-eks/lib/legacy-cluster.ts

```diff
@@ -407,6 +420,8 @@ export class LegacyCluster extends Resource implements ICluster {
 
 /**
  * Import a cluster to use in another stack
+ *
+ * @resource AWS::EKS::Cluster
```

Comment:
Is this addition because of some `jsii` validation?

Reply:
yeah jsii reports and error if I don't put it here. 
but probably I have wrong local environment, I can remove it and test inside PR build probably it's not really necessary

Reply:
updated my lcoal env. the error doesn't happen again. removed annotation